### DISCUSSION TITLE
Requires desgnintf?

### DISCUSSION THREAD

**Discussion by davidazus** [adrianosantostreina/MobilePermissions#13] [General]
I'm using Delphi 10.4 Community; when I compile this, either from GetIt or downloaded from GitHub, the compiler is popping up errors, that DesignIntf is not found. My quick Google search says DesignIntf was required for much older versions of Delphi. Anyone with success compiling recently?

**Comment by adrianosantostreina** [comment]
Hi, the component doesn't use this unit. 
On my github is the installation step by step: 
http://github.com/adrianosantostreina/MobilePermissions 

Basically need: 
1. Open Delphi in admin mode 
2. Open component package (or download by Get it) 
3. Build the package 
4. Install the package 

Check Build Target for Windows 32 

Adriano 


De: "davidazus" ***@***.***> 
Para: "adrianosantostreina" ***@***.***> 
Cc: "Subscribed" ***@***.***> 
Enviadas: Segunda-feira, 13 de setembro de 2021 0:30:25 
Assunto: [adrianosantostreina/MobilePermissions] Requires desgnintf? (Discussion #13) 





I'm using Delphi 10.4 Community; when I compile this, either from GetIt or downloaded from GitHub, the compiler is popping up errors, that DesignIntf is not found. My quick Google search says DesignIntf was required for much older versions of Delphi. Anyone with success compiling recently? 

— 
You are receiving this because you are subscribed to this thread. 
Reply to this email directly, [ https://github.com/adrianosantostreina/MobilePermissions/discussions/13 | view it on GitHub ] , or [ https://github.com/notifications/unsubscribe-auth/AFSBSUNBF3I7GX3FBOSOVMDUBVV5DANCNFSM5D43MJFA | unsubscribe ] . 


-- 


#MaiorCanalDelphiMobile 

Adriano Santos 
Tornando você um profissional mais completo 

Site oficial: [ http://adrianosantostreina.com.br/ | http://adrianosantostreina.com.br ] 
YouTube: [ http://youtube.com/adrianosantostreina | http://youtube.com/adrianosantostreina ] 
Instagram: [ http://instagram/adrianosantostreina | http://instagram/adrianosantostreina ]
